src/CMakeLists.txt

```diff
@@ -28,44 +16,7 @@ install(TARGETS simdjson-source EXPORT simdjson-config)
 # directory. It does not specify any compiler flags, even though simdjson.so/a was compiled with
 # target_link_libraries(simdjson PRIVATE simdjson-flags).
 #
-
-if(SIMDJSON_BUILD_STATIC)
-  MESSAGE( STATUS "Building a static library." )
-  add_library(simdjson STATIC "")
-else()
-  MESSAGE( STATUS "Building a dynamic library." )
-  add_library(simdjson SHARED "")
-  target_compile_definitions(simdjson INTERFACE SIMDJSON_USING_LIBRARY=1)
-  if(MSVC)
-    MESSAGE( STATUS "Building a Windows DLL using Visual Studio, exporting all symbols automatically." )
-    set_target_properties(simdjson PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS 1)
-  endif()
-endif()
-
-target_link_libraries(simdjson INTERFACE simdjson-headers) # Only expose the headers, not sources
-
-target_link_libraries(simdjson PRIVATE simdjson-source simdjson-flags)
-
-
-if(NOT MSVC)
-  ## We output the library at the root of the current directory where cmake is invoked
-  ## This is handy but Visual Studio will happily ignore us
-  set_target_properties(simdjson PROPERTIES LIBRARY_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR})
-  MESSAGE( STATUS "Library output directory (does not apply to Visual Studio): " ${PROJECT_BINARY_DIR})
-endif()
-
-#
-# Installation
-#
-install(TARGETS simdjson
-  EXPORT simdjson-config
-  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
-  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
-)
-
-install(EXPORT simdjson-config
-  FILE simdjson-config.cmake
-  NAMESPACE simdjson::
-  DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/simdjson
-)
+# TODO: For some reason simdjson-internal cannot pass checkperf if built as a static library
```

Comment:
That's weird. We've been passing checkperf in the past with static linking.

Is this happening consistently? There is some noise level in checkperf, which we've been unable to completely eliminate due to noise on the cloud-based CI machines themselves.